js/packages/core/src/utils/extract-CI-provider.ts

```diff
@@ -0,0 +1,38 @@
+export function extractCIProvider(): string {
```

Comment:
the CI is not always there, I think we should return null if we were not able to find one.
```suggestion
export function extractCIProvider(): string | null {
```